showcase/gapic-showcase/src/test/java/com/google/showcase/v1beta1/it/ITGdch.java

```diff
@@ -0,0 +1,183 @@
+package com.google.showcase.v1beta1.it;
+
+import static org.junit.Assert.*;
+
+import com.google.api.client.json.GenericJson;
+import com.google.api.client.json.JsonFactory;
+import com.google.api.client.json.gson.GsonFactory;
+import com.google.api.gax.core.FixedCredentialsProvider;
+import com.google.api.gax.core.NoCredentialsProvider;
+import com.google.api.gax.rpc.ClientContext;
+import com.google.auth.Credentials;
+import com.google.auth.oauth2.GdchCredentials;
+import com.google.showcase.v1beta1.EchoClient;
+import com.google.showcase.v1beta1.EchoSettings;
+import java.io.*;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import com.google.showcase.v1beta1.stub.EchoStubSettings;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+/**
+ * Test suite to confirm a client can be instantiated with GDCH credentials. No calls are made since
+ * it is not feasible to test against real GDCH servers (or replicate an environment)
+ */
+public class ITGdch {
+
+  private static final String TEST_GDCH_CREDENTIAL_FILE = "/test_gdch_credential.json";
+  private static final String CA_CERT_RESOURCE_PATH = "/fake_cert.pem";
+  private static final String CA_CERT_JSON_KEY = "ca_cert_path";
+  private static final String TEMP_CREDENTIAL_JSON_FILENAME = "temp_gdch_credential.json";
+
+  @Rule public TemporaryFolder tempFolder = new TemporaryFolder();
+
+  private EchoClient client;
+  private EchoSettings settings;
+  private Credentials credentials;
+
+  @Before
+  public void setup() throws IOException, URISyntaxException {
+    prepareCredentials();
+    client = null;
```

Comment:
We don't have to set it to null, the default value is null.

Reply:
Done!